main.css

```diff
@@ -0,0 +1 @@
+@import "./dist/main";
```

Comment:
@mcnuttandrew Importing this file in sass works, but not sure if it will using vanilla css, do you have an idea?